foreign.js

```diff
@@ -0,0 +1,23 @@
+const fs = require('fs');
+const path = require('path');
+const foreign = require('foreign');
+const readFile = (filename, callback) => fs.readFile(filename, {encoding: 'utf8'}, callback);
+const relative = (filename) => path.join(process.argv[2], filename);
```

Comment:
We shouldn't reference `process.argv` outside `main`. We could move this definition inside `main`.